src/CheckFor.php

```diff
@@ -210,4 +210,71 @@ public static function isType($desired_type, $row) {
     }
     return TRUE;
   }
+
+  /**
+   * Check file for redirects.
+   *
+   * @param object $row
+   *   A row object as delivered by migrate.
+   *
+   * @return mixed
+   *   string - full URL of the redirect destination.
+   *   FALSE - no detectable redirects exist in the page.
+   */
+  public static function hasHtmlRedirect($row) {
+    // Checks for meta location redirects.
+    $meta_location = qp($row)->find('meta[http-equiv="location"')->attr("url");
```

Comment:
Ok so I have rebuilt enough to get the code you needed to do the querypath part correctly.

Generally  all  `qp($row)` should get replaced with `$this->queryPath`